ADRs/RH/SIG-SRE/ADR-00000 Template.md

```diff
@@ -0,0 +1,69 @@
+# ADR-00000 Template
+
+Authors: 
+
+
+## Status
+
+&lt;_Draft, Proposed, Reviewing, Accepted, Rejected, Superseded_> 
+
+
+## Problem Statement
+
+&lt;_Why does this ADR exist? What issue is it trying to solve?_>
```

Comment:
s/What issue is it trying to solve?/What decision are you trying to reach?

For example: This document is proposing we build a hybrid cloud application that draws rainbows.

Reply:
done